lerna.json

```diff
@@ -9,7 +9,7 @@
       "allowBranch": ["main"],
       "conventionalCommits": true,
       "exact": true,
-      "message": "docs(release): design system packages\n\nskip-checks: true",
+      "message": "docs(release): design system packages\n\n[skip-ci]",
```

Comment:
`[skip-ci]` ➝ `[skip ci]`

Zie: [GitHub Actions: Skip pull request and push workflows with [skip ci]
](https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/)